### PR TITLE
Incorporate v7 standard

### PR DESCRIPTION
**Pull request by BillWagner**
Merge the `standard-v7` branch into `draft-v8`.

### CODE REVIEW

**Comment by BillWagner** [comment]
Leaving in draft status for now.
I was amazed that there weren't any conflicts. hmmmm. Because of that, I want to do more of a review than I have so far before I'm confident merging.

**Comment by RexJaeschke** [comment]
@BillWagner You have me as a reviewer, but it's not clear to me what I should look at.

**Comment by BillWagner** [comment]
Hi @RexJaeschke 

> You have me as a reviewer, but it's not clear to me what I should look at.

It's mostly to make a sanity check to see if all of V7 is in this PR. It's one merge commit, so if git did the right thing, we're fine.

**Review by RexJaeschke — APPROVED** [review]

File: admin/v8-feature-tracker.md [hunk 1]
```diff
@@ -4,30 +4,22 @@ This file identifies the known V8 features, and tracks their status.
...
-readonly instance members ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/readonly-instance-members.md)) | small | Completed | | 
-name shadowing in nested functions ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/shadowing-in-nested-functions.md)) | small | Open | | 
-static local functions ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/static-local-functions.md)) | | Almost complete | small | Pending final words for V7.0 addition of non-static local functions
-unconstrained type parameter in null coalescing operator ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/unconstrained-null-coalescing.md)) | | Completed | |
-using declarations ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/using.md)) | | Completed | small | 
-disposable ref structs/pattern-based using ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/using.md)) | | Almost complete | small | Q. on Teams re extension method support
-
+Feature | PR | Status | Effort | Annotation | Notes
```
Comment (BillWagner) [inline admin/v8-feature-tracker.md:7]: Question for @RexJaeschke 

Should I revert this change to the `v8-feature-tracker`? Is the current version in the `draft-v8` branch correct?

**Review by BillWagner — COMMENTED** [review]
I took the time to  browse all the changes and I'll mark this ready.

@RexJaeschke Can you check my one comment? There are a lot of diffs for the v8-tracker, and I don't know which branch had the correct and latest updates. 

Once you help me get the right version of the tracker, I'll merge this and start working on rebasing the PRs.

File: admin/v8-feature-tracker.md (hunk 1 above)
Comment (BillWagner) [reply admin/v8-feature-tracker.md:7]: @RexJaeschke responded offline that this change is correct. The version in v7 was more recently updated and is correct.
